docs/vickywane-ambianic-stripe-swagger-docs-0.1-resolved.yaml

```diff
@@ -0,0 +1,51 @@
+---
+openapi: 3.0.1
+info:
+  title: defaultTitle
```

Comment:
@vickywane let's use meaningful text in the Open API spec. This file will be the single source of truth for the ambianic-subscriptions API going forward.